plugins/inputs/tacacs/README.md

```diff
@@ -0,0 +1,49 @@
+# Tacacs Input Plugin
+
+The Tacacs plugin collects successful tacacs authentication response times.
```

Comment:
Is there an implementation/software we can or should reference?

Reply:
Ill try mentioning at least aruba clearpass (which we use) and the one used in tests etc